.github/workflows/edfi.database.admin package.yml

```diff
@@ -0,0 +1,107 @@
+# SPDX-License-Identifier: Apache-2.0
+# Licensed to the Ed-Fi Alliance under one or more agreements.
+# The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+# See the LICENSE and NOTICES files in the project root for more information.
+
+name: EdFi.Database.Admin Package build
+
+on:
```

Comment:
Please add a filter to monitor only .sql file changes, there's an example in the ticket description.